app/services/parser.rb

```diff
@@ -0,0 +1,75 @@
+require 'open-uri'
+
+class Parser
+  HEADLESS_RUN = true
+  DELAY_VAL    = 3
+  AVITO_SOURCE = "https://www.avito.ru/irkutsk/kvartiry/sdam/na_dlitelnyy_srok".freeze
+  class << self
+    def get_avito(count = 2)
+      run do
+        doc = Nokogiri::HTML(open(AVITO_SOURCE))
+        doc.css('h3.title a').each_with_index do |ad, index|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.